packages/cli/src/commands/generate/index.ts

```diff
@@ -15,14 +17,44 @@ export const generateSchema = async ({
     defaultSchema,
     schemas = [],
     tables = [],
+    schemaDefOverrides,
 }: GenerateSchemaOptions) => {
-    const result = await generate({
-        defaultSchema,
-        connectionString,
-        includeSchemas: schemas,
-        includeTables: tables,
-        outDir: out,
-    });
+    if (schemaDefOverrides === undefined) {
+        try {
+            return await generate({
+                defaultSchema,
+                connectionString,
+                includeSchemas: schemas,
+                includeTables: tables,
+                outDir: path.resolve(path.join(process.cwd(), out)),
+            });
+        } catch (error) {
+            console.log('Error: ', JSON.stringify(error, null, 2));
+        }
+    } else if (isValidSchemaDefOverrides(schemaDefOverrides)) {
+        try {
+            return await generate({
+                defaultSchema,
+                connectionString,
+                includeSchemas: schemas,
+                includeTables: tables,
+                schemaDefOverrides,
+                outDir: path.resolve(path.join(process.cwd(), out)),
+            });
+        } catch (error) {
+            console.log('Error: ', JSON.stringify(error, null, 2));
+        }
+    } else {
+        const lines = [
+            'Invalid shape of schema def overrides:',
+            '',
+            `${JSON.stringify(schemaDefOverrides, null, 2)}`,
+        ];
 
-    return result;
+        throw new Error(lines.join('\n'));
+    }
 };
+
+function isValidSchemaDefOverrides(value: any): value is SchemaDefOverrides {
+    return value satisfies SchemaDefOverrides;
```

Comment:
this function is not doing anything at runtime. This function is literally equivalent to 
```ts
function isValidSchemaDefOverrides(x) {return x}
```